CRM/Contact/BAO/Query.php

```diff
@@ -4916,7 +4918,7 @@ public function searchQuery(
     //      MySQL expect the columns present in GROUP BY, must be present in SELECT clause and that results into error, needless to have other columns.
     //   2. When GROUP BY columns are present then disable FGB otherwise it demands to add ORDER BY columns in GROUP BY and eventually in SELECT
     //     clause. This will impact the search query output.
-    $disableFullGroupByMode = ($sortByChar || !empty($groupByCols) || $groupContacts);
+    $disableFullGroupByMode = ($sortByChar || !empty($groupBy) || $groupContacts);
```

Comment:
this has the same effect - I want to extract the above so less functions = better